main.py

```diff
@@ -33,42 +47,96 @@ def main():
 
     for pr in prs:
         print(f"\nClassifying files from PR {pr} for predictions training ")
-        processing.process_files(
-            ai, db, pr
-        )  # Here is where ASTs and classification is done. All the "heavy lifting" of the core engine
-
-    db.save()
-    db.close()
-
-    # Pseudo code below... just the concepts (and thoughts, this is not set in stone... what do you think?)
-    # import llm_classifier
-    # training_model == (get from command line)
-    #
-    # if training_model == "GPT":
-
-    #     # Generate fine tuning file
-    #
-    #     data_from_extraction = database manager query OutputTable
-    #
-    #     system_message, assistant_message = llm_classifier.generate_system_message(domain_dictionary, data_from_extraction)
-    #     generate_gpt_messages(system_message, assistant_message, df)
-
-    #     # Fine tune GPT Model
-    #     llm_classifier = fine_tune_gpt(openAI_key)
-    #     save_model(llm_classifier)
 
-    # if model == "Random Forest":
+        # Here is where ASTs and classification are done;
+        # all the "heavy lifting" of the core engine
+        processing.process_files(ai, db, pr)
 
-    #     data_from_extraction = database manager query OutputTable
-    #
-    #     rf_model = llm_classifier.train_rf(domain_dictionary, data_from_extraction)
-
-    #     save_model(rf_model)
+    db.save()
 
-    # exit()
+    method = cfg_dict["classification_method"]
```

Comment:
The comment below could also be applied here, which is something I contributed. We could probably shorten this to

```suggestion
    method = cfg_dict["clf_method"]
```